<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_raton_ingresar</class>
 <widget class="QMainWindow" name="vtn_raton_ingresar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>214</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Raton</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_ingreso_raton">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingresar Rat√≥n</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_marca">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_entrada">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo Entrada:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_marca">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_entrada">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_raton_ingresar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cantidad_botones">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_entrada_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad Botones</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
